CMSgov/onemac
# OY2-22026 - Update waiver amendment

Story: https://qmacbis.atlassian.net/browse/OY2-22026
Endpoint: See github-actions bot comment

### Details

Update header label and add additional fields to amendment detail page.

### Changes

- update waiverAmendmentDetail.tsx

### Implementation Details

- ACs mention left hand nav changes and update of Date Submitted to Initial Submission Date but those two items should be handled by their respective stories also in this sprint

### Test Plan

1. Login as state submitter
2. Navigate to the detail page for a waiver amendment
3. verify heading change
4. verify removal of waiver title
5. verify addition of new data elements
